docs/usage.rst

```diff
@@ -39,3 +48,56 @@ documentation: :py:func:`tedana.workflows.t2smap_workflow`.
    :ref: tedana.workflows.t2smap._get_parser
    :prog: t2smap
    :func: _get_parser
+
+
+.. _constructing ME-EPI pipelines:
+
+Constructing ME-EPI pipelines
+-----------------------------
+
+``tedana`` must be called in the context of a larger ME-EPI preprocessing pipeline.
+Two common pipelines which support ME-EPI processing include `fMRIPrep`_ and `afni_proc.py`_.
+
+.. _fMRIPrep: https://fmriprep.readthedocs.io
+.. _afni_proc.py: https://afni.nimh.nih.gov/pub/dist/doc/program_help/afni_proc.py.html
+
+Users can also construct their own preprocessing pipeline for ME-EPI data from which to call ``tedana``.
+There are several general principles to consider when constructing ME-EPI processing pipelines
+for use with ``tedana``:
```

Comment:
Can you add in a single line that summarizes preprocessing around tedana? Something like:

Generally, we recommend performing slice timing correction and motion correction **before** tedana, and performing distortion correction, spatial normalization, smoothing, and any rescaling or filtering **after** tedana.

Reply:
Done ! Thanks for the feedback, @tsalo !! :sparkling_heart:

Reply:
Awesome. Looks great!